css/style.css

```
@@ -312,6 +322,20 @@ a:visited {
   fill-opacity: 0.8;
 }
 
+.social-icons {
+  margin: 0 0 0 27px;
+  padding: 0;
+  display: grid;
+  grid-auto-flow: column;
```

```suggestion
  grid-template-columns:  1fr 1fr 1fr 1fr;
  grid-template-rows: auto;
```
ты авто-флоу использовала не по-назначению